src/Components/Ignitor/src/BlazorClient.cs

```diff
@@ -233,7 +236,7 @@ private async Task<CapturedRenderBatch> WaitForRenderBatch(TimeSpan? timeout = n
                 {
                     return await PrepareForNextBatch(timeout ?? DefaultOperationTimeout);
                 }
-                catch (OperationCanceledException)
+                catch (TimeoutException) when (FormatError != null)
```

Comment:
This was a pain to discover: `FomatError` isn't set up in the benchmarks repo. Took me forever to figure out was causing a null ref.

Reply:
@pranavkm What is format error?

Reply:
It's a callback used in the test to pretty print the exception + log: https://github.com/aspnet/AspNetCore/blob/master/src/Components/test/E2ETest/ServerExecutionTests/IgnitorTest.cs#L59-L63.